multiraft/multiraft.go

```diff
@@ -428,14 +436,18 @@ type state struct {
 	groups    map[proto.RangeID]*group
 	nodes     map[proto.RaftNodeID]*node
 	writeTask *writeTask
+	// buffer the event if s.Events channel is full to avoid the deadlock
+	// between s.Events channel and callbackChan.
+	pendingEvents []interface{}
```

Comment:
Cool, sending a slice of events is a good solution. Since the channel is unbuffered. I think we can just remove the comment here - there's no longer any question of "if the s.Events channel is full"